testing/fuzzing/toxsave_harness.cc

```diff
@@ -16,9 +16,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     assert(tox_options != nullptr);
     assert(error_options == TOX_ERR_OPTIONS_NEW_OK);
 
-    uint64_t clock = 0;
-    auto sys = fuzz_system(clock);
-    tox_options_set_operating_system(tox_options, sys.get());
+    Fuzz_Data input{data + size, 0};  // empty data, since we use it all for savedata.
```

Comment:
Why point at `data + size`? To trigger ASAN on read? How about a dedicated constructor for this case?

Reply:
Because semantically, we create one Fuzz_Data and then consume all of its input, which results in this. I can write that explicitly instead I guess.

Reply:
makes sense, it's just a bit confusing because the order is reversed in this code.